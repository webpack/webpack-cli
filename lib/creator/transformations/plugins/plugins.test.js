const defineTest = require('../../../transformations/defineTest');

defineTest(__dirname, 'plugins', 'plugins-0', {plugins : [
	'new webpack.optimize.CommonsChunkPlugin({name:' + '\'' + 'vendor' + '\'' + ',filename:' + '\'' + 'vendor' + '-[hash].min.js\'})'
]});
