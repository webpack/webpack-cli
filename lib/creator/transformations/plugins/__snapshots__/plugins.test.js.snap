// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plugins transforms correctly using "plugins-0" data 1`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    plugins: [
        new webpack.optimize.CommonsChunkPlugin({name:'vendor',filename:'vendor-[hash].min.js'})
    ]
}
"
`;
